File: Iniciante/URI 1930.lua
line = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line[n] = i
end
T1 = tonumber(line[1])
T2 = tonumber(line[2])
T3 = tonumber(line[3])
T4 = tonumber(line[4])

totalTomadas = (T1 + T2 + T3 + T4) - 3
print(totalTomadas)

File: Iniciante/URI 1930.clj
(require '[clojure.string :as str])

(def line (str/split (read-line) #" "))
(def T1 (Integer/parseInt (get line 0)))
(def T2 (Integer/parseInt (get line 1)))
(def T3 (Integer/parseInt (get line 2)))
(def T4 (Integer/parseInt (get line 3)))

(def totalTomadas(- (+ T1 T2 T3 T4) 3))

(println totalTomadas)